src/include/mom_func.h

```diff
@@ -242,7 +242,6 @@ typedef enum {
 	PRE_FINISH_FAIL,
 	PRE_FINISH_FAIL_JOB_SETUP_SEND,
 	PRE_FINISH_FAIL_JOIN_EXTRA,
```

Comment:
Remove the comma at the end of the line. It is now the last entry.

Reply:
nice catch.  Done.